src/router/schema.ts

```diff
@@ -56,18 +52,54 @@ export async function compose(req, res) {
 }
 
 export async function push(req, res) {
-	const service = Joi.attempt(
+	const body = Joi.attempt(
 		req.body,
 		Joi.object().keys({
 			name: Joi.string().min(3).max(200).required(),
 			version: Joi.string().min(1).max(100).required(),
 			type_defs: Joi.string().required(),
 			url: Joi.string().uri().min(1).max(255).allow(''),
+			usage_days: Joi.number().max(MIN_DAYS).optional().default(MIN_DAYS),
+			min_usages: Joi.number()
+				.min(MIN_USAGES)
+				.optional()
+				.default(MIN_USAGES),
 		})
 	);
 
+	const service = (({ name, version, type_defs, url }) => ({
```

Comment:
I am not sure we want this control in the `push`  as  if at some point there is a need to do it , let me ask for it and I will let you know if we keep it or not :)

Reply:
Nice :)

Reply:
Decided, we keep it only in the `diff` thank you!